app/crate/db/shows.py

```
@@ -140,3 +219,41 @@ def get_attending_show_ids(user_id: int, show_ids: list[int]) -> set[int]:
+
+
+def get_show_reminders(user_id: int, show_ids: list[int] | None = None) -> list[dict]:
+    with get_db_ctx() as cur:
+        if show_ids:
+            placeholders = ",".join(["%s"] * len(show_ids))
+            cur.execute(
+                f"""
+                SELECT id, user_id, show_id, reminder_type, created_at, triggered_at
+                FROM user_show_reminders
+                WHERE user_id = %s AND show_id IN ({placeholders})
+                """,
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)